tfx/examples/cifar10_pipeline/README.md

```diff
@@ -0,0 +1,54 @@
+# CIFAR 10 Example
+
+The CIFAR 10 example demonstrates the end-to-end workflow and steps of how
+to analyze, validate and transform data, train a model, analyze and serve it. It
+uses the following [TFX](https://www.tensorflow.org/tfx) components:
+
+* [ExampleGen](https://github.com/tensorflow/tfx/blob/master/docs/guide/examplegen.md)
+  ingests and splits the input dataset.
+* [StatisticsGen](https://github.com/tensorflow/tfx/blob/master/docs/guide/statsgen.md)
+  calculates statistics for the dataset.
+* [SchemaGen](https://github.com/tensorflow/tfx/blob/master/docs/guide/schemagen.md)
+  SchemaGen examines the statistics and creates a data schema.
+* [ExampleValidator](https://github.com/tensorflow/tfx/blob/master/docs/guide/exampleval.md)
+  looks for anomalies and missing values in the dataset.
+* [Transform](https://github.com/tensorflow/tfx/blob/master/docs/guide/transform.md)
+  performs feature engineering on the dataset.
+* [Trainer](https://github.com/tensorflow/tfx/blob/master/docs/guide/trainer.md)
+  trains the model using TensorFlow [Estimators](https://www.tensorflow.org/guide/estimators)
+* [Evaluator](https://github.com/tensorflow/tfx/blob/master/docs/guide/evaluator.md)
+  performs deep analysis of the training results.
+* [ModelValidator](https://github.com/tensorflow/tfx/blob/master/docs/guide/modelval.md)
+  ensures that the model is "good enough" to be pushed to production.
+* [Pusher](https://github.com/tensorflow/tfx/blob/master/docs/guide/pusher.md)
+  deploys the model to a serving infrastructure.
+
+## The dataset
```

Comment:
Could you also add more documentation on how to get the training data in tfrecord format?

Reply:
This is tricky. i got this data with some scripts I wrote on the fly. This is because I didn't find a clean way yet to import data from various sources with a custom Executor without the need of building a spcific Docker image that is really something complicated for a newcomer playing with the example. Any suggestion is welcome.

Reply:
I see. It's ok, we will try to fix that.